docs/self-managed/components/modeler/web-modeler/overview.md

```diff
@@ -2,7 +2,30 @@
 id: overview
 title: "Web Modeler on Self-Managed"
 sidebar_label: "Overview"
-description: "About Web-Modeler on Self-Managed"
+description: "About Web Modeler on Self-Managed"
 ---
 
-About Web-Modeler on Self-Managed
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import AoGrid from '/docs/components/react-components/\_ao-card';
+import { a2aConnectorCards } from '/docs/components/react-components/\_ao-card-data';
+
+Web Modeler on Self‑Managed lets you run Camunda's modeling experience in your own infrastructure.
```

Comment:
Same comment here -- I think "Web Modeler _on_ Self-Managed" sounds weird based on how we phrase this elsewhere in the docs.